src/cli/README.md

```diff
@@ -2864,13 +2864,26 @@ Done
 
 ### networkdiagnostic reset \<addr\> \<type\> ..
 
-Send network diagnostic request to reset \<addr\>'s tlv of \<type\>s. Currently only `MAC Counters`(9) is supported.
+Send network diagnostic request to reset \<addr\>'s tlv of \<type\>s. Currently `MAC Counters`(9) is supported.
 
 ```bash
 > diagnostic reset fd00:db8::ff:fe00:0 9
 Done
 ```
 
+### networkdiagnostic nonprfchannelmask
+
+Get or set the non-preferred channel mask value used to respond to a Network Diagnostics Get request for the corresponding TLV.
+
```

Comment:
Should add here a line saying that the value format is uint32. (Since this isn't defined by the Thread spec, it may not be obvious what kind of value needs to be passed.)

Reply:
Added more details on the mask format here.